addons/mail/models/mail_thread.py

```diff
@@ -3401,3 +3403,14 @@ def _get_mail_thread_data(self, request_list):
         if 'suggestedRecipients' in request_list:
             res['suggestedRecipients'] = self._message_get_suggested_recipients()[self.id]
         return res
+
+    def _related_res_partner_default_create_values(self, normalized_email):
```

Comment:
Mmmh, I think we should try to find a business name to that method. What about something like "_get_customer_information" or "_get_customer_create_values" ? Purpose is to have something customer oriented (customer = partner_id field generally). As we have some code dealing with the specific partner_id, it would makes sense to extract it / improve it in a mixin in a (near) future, so already having this "customer" keyword makes sense to me. What do you think ?

Reply:
As we currently limit the information extracted to customer, it is indeed a good idea. I have changed it to _get_customer_information and updated the doc.